modules/integration_aws-backup/variables.tf

```diff
@@ -0,0 +1,3 @@
+variable "aws_account_id" {
```

Comment:
I don't think forcing aws_account_id is a good idea, not everyone wants to filter by aws_account_id
It's best to let people choose if they want to by setting the appropriate filter in their own terraform

Reply:
Hello, we don't have much choice because recovery points are not automatically tagged according to the convention used with the detectors (see comment in `common-filters.tf`).

Reply:
Variable removed after further discussions.